src/Build/Resources/Strings.resx

```diff
@@ -1353,7 +1353,7 @@
     <comment>{StrBegin="MSB4209: "}</comment>
   </data>
   <data name="AffinityConflict" xml:space="preserve">
-    <value>MSB4213: The specified request affinity {0} conflicts with a previous affinity {1} specified for this project.</value>
+    <value>MSB4213: The specified request affinity {0} conflicts with a previous affinity {1} specified for project {2} with global properties {3}</value>
```

Comment:
We have passed the loc deadline for 16.11. This looks like a nice improvement but can it be pushed out to 17.0?

Reply:
Yup, I'll cherry-pick it out into a PR to main.